webui/react/src/utils/styled.ts

```diff
@@ -6,3 +6,5 @@ export const isPropTrue = (props: Record<string, any>, propName: string): boolea
   }
```

Comment:
nit: I just checked and this file isn't used anymore. I just forgot to remove this during css-modules migration. Feel free to remove this file.

Reply:
removed